youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -431,21 +431,36 @@ def run(self, information):
 
 
 class FFmpegMetadataPP(FFmpegPostProcessor):
+
+    def __init__(self, downloader=None, preferredinfo=None):
+        super(FFmpegMetadataPP, self).__init__(downloader)
+        self._preferredinfo = preferredinfo if isinstance(preferredinfo, dict) else None
+
     def run(self, info):
         metadata = {}
 
+        def add_info(meta_list, info_list, metadata, info_obj):
+            if info_obj is not None:
+                for info_f in info_list:
+                    if info_obj.get(info_f) is not None:
+                        for meta_f in meta_list:
+                            metadata[meta_f] = info_obj[info_f]
+                        return info_f
+            return None
+
         def add(meta_list, info_list=None):
             if not info_list:
                 info_list = meta_list
             if not isinstance(meta_list, (list, tuple)):
                 meta_list = (meta_list,)
             if not isinstance(info_list, (list, tuple)):
                 info_list = (info_list,)
-            for info_f in info_list:
-                if info.get(info_f) is not None:
-                    for meta_f in meta_list:
-                        metadata[meta_f] = info[info_f]
-                    break
+            preferred_key = add_info(meta_list, info_list, metadata, self._preferredinfo)
+            if preferred_key is None:
+                add_info(meta_list, info_list, metadata, info)
+            else:
+                for info_f in info_list:
+                    info[info_f] = self._preferredinfo[preferred_key]
```

Comment:
1. No.
2. This won't have any effect anyway since post processors operate on info dict copy.